app/controllers/content_files_controller.rb

```diff
@@ -2,15 +2,19 @@
 
 # Controller for a Work content file
 class ContentFilesController < ApplicationController
-  # Skipping authorization for now
-  skip_verify_authorized only: %i[show edit update destroy]
```

Comment:
Can you replace the changes to the action methods below with:

```
  verify_authorized only: %i[show edit update destroy]
```

❓

Reply:
We already do that here: https://github.com/sul-dlss/hungry-hungry-hippo/blob/main/app/controllers/application_controller.rb#L11

`verify_authorized` makes sure that `authorize!` is called; it doesn't do the calling itself.

Reply:
Ah, thank you!